src/map/clif.c

```diff
@@ -18155,21 +18155,30 @@ static void clif_parse_Adopt_reply(int fd, struct map_session_data *sd)
 ///     1 = Boss is alive (position update) (BOSS_INFO_ALIVE).
 ///     2 = Boss is alive (initial announce) (BOSS_INFO_ALIVE_WITHMSG).
 ///     3 = Boss is dead (BOSS_INFO_DEAD).
-static void clif_bossmapinfo(int fd, struct mob_data *md, short flag)
+static void clif_bossmapinfo(struct map_session_data *sd, struct mob_data *md, short flag)
 {
+	int fd = sd->fd;
```

Comment:
missing nullpo check for ``sd``
and probably better use ``const int fd = sd->fd``

Reply:
Actually I don't see a reason to change the parameter from `int fd` to `struct map_session_data *sd` since `sd` is only used to set `fd`...